sdf.md

```diff
@@ -485,14 +485,80 @@ complex devices that will use one or more `sdfObject` definitions.
 A definition in an `sdfThing` group can refine the metadata of the definitions it
 is composed from: other definitions in `sdfThing` groups definitions in `sdfObject` groups.
 
+## Member names: Given Names and Quality Names
+
+SDF models are JSON objects (maps) that mostly employ JSON objects as
+member values, which in turn mostly employ JSON objects as their
+member values, and so on.
+This nested structure of JSON objects creates a tree, where the edges
+are the member names (map keys) used in these JSON objects.
+(In certain cases, where member names are not needed, JSON arrays may
+be interspersed in this tree.)
+
+### Given Names and Quality Names
+
+For any particular JSON object in an SDF model, the set of member
+names that are used is either of:
+
+* A set of "*Quality Names*", where the entries in the object are
+  Qualities.  Quality Names are defined by the present specification
+  and its extensions, together with specific semantics to be
+  associated with the member value given with a certain Quality Name.
+
+* A set of "*Given Names*", where the entries in the object are separate
+  entities (definitions, declarations, etc.) that each are freely named by
```

Comment:
"freely named" sounds quite liberal. I wonder if just "named" is better?

Reply:
The point is that the model creator is free to choose the name, so "freely" was trying to emphasize that.   What is wrong about being liberal here?

Reply:
"without constraint"?